docs/tutorial/01_bootstrap_juju.md

```diff
@@ -25,6 +25,13 @@ From your terminal, install Juju.
 sudo snap install juju --channel=3.1/stable
 ```
 
+Add your user to the `snap_microk8s` group:
+
+```console
+sudo usermod -a -G snap_microk8s $USER
```

Comment:
Those should probably be in the `install MicroK8s` section (just before the `microk8s enable ...`